components/Common/Breadcrumbs/index.tsx

```diff
@@ -28,6 +29,13 @@ const Breadcrumbs: FC<BreadcrumbsProps> = ({
   const lengthOffset = maxLength - totalLength;
   const isOverflow = lengthOffset < 0;
 
+  const formatLabelWithWordCount = (label: string, maxWords: number) => {
```

Comment:
JavaScript is not needed for this. We can achieve this with CSS ellipsis

https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow

Reply:
`text-ellipsis` https://tailwindcss.com/docs/text-overflow

Reply:
will proceed with the change immediately